aces/Loris
# [NDB_BVL_Instrument] Hide inactive examiners for site from selection, unless if already selected

## Brief summary of changes

- Hide examiners from an instrument if they are inactive for the site
- Does not hide them if they were already selected

#### Testing instructions (if applicable)

1. Update an examiner in examiners_psc_rel to have active = 'N'
2. Try to set them as examiner for an instrument, see that they do not appear
3. Set them as someone else, then set them to active = 'N' and see that they are still there

[CCNA OVERRIDE PR ](https://github.com/aces/CCNA/pull/7127)
